SPECS/redis/CVE-2020-14147.patch

```diff
@@ -0,0 +1,41 @@
+diff --git a/deps/lua/src/lua_struct.c b/../redis/deps/lua/src/lua_struct.c
+index 4d5f027..c58c8e7 100644
+--- a/deps/lua/src/lua_struct.c
++++ b/../redis/deps/lua/src/lua_struct.c
```

Comment:
Under %prep, you have "%patch1 -p1". "p1" strips the first leading slash, so to apply this patch, I believe you need:

+++ b/deps/lua/src/lua_struct.c

Reply:
Fixed